index.html

```diff
@@ -151,11 +152,40 @@ <H3> About </H3>
 	}
 
 	function parseDate(date){
-		var year = Math.floor(date/10000);
```

Comment:
Why did you delete these!!! I told you they are very important...

Reply:
Look down about 20 lines, they are still there slightly modified